cypress/integration/ete/registration/register.cy.ts

```diff
@@ -209,7 +209,7 @@ describe('Registration flow', () => {
         expect(body).to.have.string('This account already exists');
         expect(body).to.have.string('Sign in');
         expect(body).to.have.string('Reset password');
-        expect(body).to.have.string('This link is only valid for 30 minutes.');
+        expect(body).to.have.string('This link is only valid for 60 minutes.');
```

Comment:
following a discussion yesterday we decided to drop the "only" from the message copy now the time is longer than 30 mins, so it should now read:

"This link is valid for 60 minutes."

Reply:
I'm not sure what is so wrong about saying that it's "only" valid for 60 minutes that would justify changing it everywhere but if the decision is already made I'll change it

Reply:
I'm not really sure how the "changes requested" feature works...
I made the required changes, do you have to click something to make it official that your requested changes were made ?

Reply:
you can re-request a review, or like i'm about to do i can just put a new review through myself 👍